Create VM from Azure Image Step Fails if image name is provided. Need…

maniSbindra opened the pull request:
**Create VM from Azure Image** Step Fails if **image name** is provided as currently shown in docs. The **full image id** needs to be provided.
i.e. Following command Fails
```
az vm create \
    --resource-group myResourceGroup \
    --name myVM \
    --image myPackerImage \
    --admin-username azureuser \
    --generate-ssh-keys 
```
 The Error message is 
```
Invalid image "managedPackerLinux". Use a custom image name, id, or pick one from ['CentOS', 'CoreOS', 'Debian', 'openSUSE-Leap', 'RHEL', 'SLES', 'UbuntuLTS', 'Win2016Datacenter', 'Win2012R2Datacenter', 'Win2012Datacenter', 'Win2008R2SP1']
```

When Full Image Id is provided as shown in command command below it succeeds (subcriptiontionid, resourcegroupname and imagename need to be changed)
```azurecli
az vm create \
    --resource-group myResourceGroup \
    --name myVM \
    --image "/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>/providers/Microsoft.Compute/images/<image-name>"  \
    --admin-username azureuser \
    --generate-ssh-keys
```

azurecla commented:
Hi __@RyKilleen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@RyKilleen : Thanks for your contribution to the Azure documentation! The author, @rmcmurray, has been notified to review your proposed change.

rmcmurray commented:
Thanks, @RyKilleen - that's a good suggestion, we should include that. #sign-off

maniSbindra committed:
Create VM from Azure Image Step Fails if image name is provided. Need to Provide Image id

Following command Fails
```
az vm create \
    --resource-group myResourceGroup \
    --name myVM \
    --image myPackerImage \
    --admin-username azureuser \
    --generate-ssh-keys 
```
 The Error message is 
```
Invalid image "managedPackerLinux". Use a custom image name, id, or pick one from ['CentOS', 'CoreOS', 'Debian', 'openSUSE-Leap', 'RHEL', 'SLES', 'UbuntuLTS', 'Win2016Datacenter', 'Win2012R2Datacenter', 'Win2012Datacenter', 'Win2008R2SP1']
```

When Full Image Id is provided command succeds
```azurecli
az vm create \
    --resource-group myResourceGroup \
    --name myVM \
    --image "/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>/providers/Microsoft.Compute/images/<image-name>"  \
    --admin-username azureuser \
    --generate-ssh-keys
```
